### ISSUE TITLE
Footer Enhancement

### ISSUE THREAD

**Issue by MehakMantri** [camicroscope/camicroscope.github.io#30]
## Description

**Current Footer:**
The current footer of the website contains only a microscope image.

**Suggested Changes:**
I propose enhancing the footer by adding a "Join Our Community" section and including a copyright notice.

### Join Our Community

Become a part of our open-source community and contribute to the dynamic evolution of our platform. Join now to collaborate with other enthusiasts, share your ideas, and contribute to cutting-edge advancements in medical imaging technology.

[Join Now](#community-join-link)

### Copyright

&copy; 2023 caMicroscope. All rights reserved.

## Rationale

The proposed changes aim to engage users by inviting them to join the community and contribute to the platform's development. Additionally, the copyright notice helps communicate ownership and protects intellectual property.

##screenshot

![image](https://github.com/camicroscope/camicroscope.github.io/assets/114497437/13aae00d-1494-4498-89ec-ca68f9fa5af8)

Thank you for considering these enhancements to the website footer! I would like to work on the same

**Comment by birm** [comment]
We already have a copyright notice; are you by chance using noscript?

**Comment by MehakMantri** [comment]
no, I am not suing script... I am suggesting adding this in the footer section of the website

**Comment by MehakMantri** [comment]
okay..do you want me to add "join your community" link in footer?
